src/actions/guides/getting-started/starting_project.cr

```diff
@@ -28,8 +28,26 @@ class Guides::GettingStarted::StartingProject < GuideAction
     * No static file serving or public folder
     * No HTML rendering folders
 
-    > If you'd like to skip the wizard, you can run `lucky custom.init my_app`. You can
-    > pass `--api` or `--no-auth` as options. Use the `-h` flag for more help.
+    ## Skipping the Wizard
+
+    If you'd like to skip the wizard, you can run `lucky custom.init my_app`. This option supports
+    all of the same options as the wizard does, but also with some additional flags for quick customization.
+
+    This option is great if you need to generate a Lucky app programmatically, or just prefer to get going right away.
+
+    ```
+    # Generate an API only app
```

Comment:
I think showing with just project name here would be great. A lot of people skip to the code section

Reply:
You mean an example of just `lucky custom.init my_app`?

Reply:
Yep! I know you have it above but having it here with the others would be helpful I think. A lot of people skip straight to the high contrast code sample